common/views/components/PopupDialog/PopupDialog.tsx

```diff
@@ -147,19 +147,18 @@ const PopupDialogCTA = styled(Space).attrs({
   },
   className: `${font('intb', 5, { small: 3, medium: 3 })} rounded-corners`,
 })`
-  display: inline-block;
```

Comment:
Why have you removed this line?

Reply:
I'm really unsure, I wonder if it's a merge conflict leftover 🤔 I'll dig.

Reply:
No idea why and actually thanks, this showed me that on the line just after, the background color was wrong ! (set to `purple` instead of `accent.purple`). Guess I changed the wrong something in that file.